File: js/ejer53.js
//registrar usuarios (nommbre y edad ) en arreglo
let usuarios = [];

function registrarUsuario() {
    let nombre = prompt("Ingrese el nombre del usuario:");
    let edad = parseInt(prompt("Ingrese la edad del usuario:"));    
    usuarios.push({ nombre, edad });
}

function mostrarUsuarios() {
    document.write("Usuarios registrados:");
    usuarios.forEach((usuario, index) => {
        document.write(`${index + 1}. Nombre: ${usuario.nombre}, Edad: ${usuario.edad}`);
    });
}
// Ejemplo de uso
registrarUsuario();
registrarUsuario();
mostrarUsuarios();

File: python/ejer53.py
#registrar usuarios (nommbre y edad ) en arreglo
usuarios = []

def registrar_usuario(nombre, edad):
    usuarios.append({'nombre': nombre, 'edad': edad})

def mostrar_usuarios():
    for usuario in usuarios:
        print(f"Nombre: {usuario['nombre']}, Edad: {usuario['edad']}")
# Ejemplo de uso
registrar_usuario("Alice", 30)
registrar_usuario("Bob", 25)
mostrar_usuarios()  

